src/main/java/mSearch/daten/ListeFilme.java

```diff
@@ -313,19 +307,21 @@ public synchronized ListeFilme neueFilme(ListeFilme orgListe) {
         return ret;
     }
 
-    public String getFileSizeUrl(String url, String sender) {
+    //FIXME bring to DatenFilm and reduce calculation
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Take the required action to fix the issue indicated by this comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1134)